src/scripts/Player.js

```diff
@@ -1,24 +1,28 @@
 import $ from 'jquery';
 
 class Player {
-  constructor() {
+  constructor(name, playerNum) {
     this.totalPoints = 0;
-    this.name;
-    this.playerNum;
+    this.name = name;
+    this.playerNum = playerNum;
     this.incorrectGuesses = [];
-    this.multiplier;
+    this.multiplier = 0;
   }
-  makeGuess() {
-
+  makeGuess(playerGuess, survey) {
```

Comment:
Nice work with this implementation, looks very clean. Functionality is nicely singly responsible.